Character_Builder/models.py

```diff
@@ -35,14 +35,53 @@
 # As this is in the character builder folder, this will focus on
 # the character information 
 
+# Automatically creates CharacterIDs
+def newCharacterID():
+    previousCharacter = Character.objects.last()
+    if not previousCharacter is None:
+        newID = previousCharacter.characterID + 1
+    else:
+        newID = 0
+
+    return newID
+
+# Automatically creates AbilityScoreSetIDs
+def newAbilityScoreSetID():
```

Comment:
As Ben said, it may be optimal to use the built in function to create a primary key, or to simply use the built in .id field every model has.

To make your own custom primary key:
models.AutoField(primary_key=True)